docs/components/links/README.md

```diff
@@ -166,6 +186,36 @@ Display icon on right of link message.
 ```
 
 </cdr-doc-example-code-pair>
+
+## Accessibility
+
+To ensure that usage of this component complies with accessibility guidelines:
+
+- Always use a `<button>` element via the `tag` prop when there is no `href` attribute that can be applied to the link. Examples are:
+  - Toggling a display to full screen
+  - Opening a modal window
+  - Triggering a popup menu
+  - Playing media content
+- Always use the default `<a>` element for a link when the link will navigate the user to the location specified by the `href` attribute
+- Avoid using only “click here” or “start here” but if screen space for text is minimal:
+  - Provide text that can be read by screen readers
+  - Use an inline element for hidden text with the 'cdr-sr-only' class
+
+```vue
+  <cdr-link>
+    Start here <span class="cdr-sr-only">for help finding the proper sleeping bag </span>
+  </cdr-link>
+```
+
+- Ensure assistive technology can find all links on a page:
+  - Always provide a `href` attribute. Empty `href` attributes are not considered true links
+  - Ensure that each link can be accessed via the keyboard, don’t manipulate the default tab index
+  - When using images as links, the `<alt>` tag acts as the link text. Describe the action or hyperlink that the image represents
+
```

Comment:
Can you add a line break before the last section in Accessibility similar to Headings and Paragraphs?